spec/features/users/01_user_create_account_spec.rb

```diff
@@ -0,0 +1,68 @@
+require 'rails_helper'
+
+feature "user create account", %{
+  As a first-time unauthenticated user
+  I want to sign up
+  So that I can use the rest of the features.
+} do
+  # Acceptance Criteria
+  # [ ] Link to sign up exists in the nav-bar when not logged in
+  # [ ] Sign-up page has a form
+  # [ ] Form returns error when required fields are not entered correctly
+  # [ ] Displays success message and redirects to user profile when form
+  #     is submitted correctly
+
+  scenario 'user sees sign-up registration Title' do
+    visit root_path
+    click_link("Sign Up")
+
+    expect(page).to have_content("Sign up")
+  end
+
+  scenario 'user sees sign-up form' do
+    visit root_path
+    click_link("Sign Up")
+
+    expect(page).to have_css("form#new_user")
```

Comment:
I mentioned this on another recent PR, it's not really required to test if there is a form for a feature test if the form being validly filled out happens for the same feature. It's sort of redundantly testing, which can create a long running test suite. There's no hard line of what should and shouldn't be tested, it's something you practice and get a feel for with time